src/components/Skeleton/Skeleton.module.css

```diff
@@ -0,0 +1,35 @@
+/*------------------------------------*\
+    # SKELETON
+\*------------------------------------*/
+
+@keyframes bg-wipe {
+  0% {
+    background-position: 0% 0%;
+  }
+  100% {
+    background-position: 100% 0%;
+  }
+}
+
+/**
+ * Skeleton
+ */
+.skeleton {
+  border-radius: var(--eds-border-radius-lg);
+  background-color: var(--eds-theme-color-background-neutral-default);
+  background-image: linear-gradient(
+    to right,
+    var(--eds-theme-color-background-neutral-subtle) 50%,
+    var(--eds-theme-color-background-neutral-default) 65%,
+    var(--eds-theme-color-background-neutral-subtle) 80%
+  );
+  background-size: 500% 100%;
+}
+
+.skeleton--is-animating {
+  animation: bg-wipe 2s ease infinite;
```

Comment:
Is there a particular reason you chose `ease`? We usually use `ease-out`.

Reply:
@dierat not really. lemme see how `ease-out` looks tho (i usually use `ease-in-out` and i have been known to do my own curve when i'm feeling really energized)

Reply:
I'm not really opinionated by which one we use, but I feel like it's nice to keep them consistent unless there's a specific reason why an animation needs a different timing function.